src/main/webapp/my-lists.html

```diff
@@ -0,0 +1,137 @@
+                                <label for="list-topic-select">Topic:</label>
+                                <select name="topic" id="list-topic-select" class="form-control" onchange="checkOtherSelected(this.value)" required>
+                                    <!-- <option selected disabled>Topic</option> -->
+                                    <option value="math">Math</option>
+                                    <option value="physics">Physics</option>
+                                    <option value="chemistry">Chemistry</option>
+                                    <option value="biology">Biology</option>
+                                    <option value="computer science">Computer Science</option>
+                                    <option value="social studies">Social Studies</option>
+                                    <option value="english">English</option>
+                                    <option value="spanish">Spanish</option>
+                                    <option value="french">French</option>
+                                    <option value="chinese">Chinese</option>
+                                    <option value="other">Other</option>
+                                </select>
+                            </div>
+                            <div style="float:right" class="form-group mx-sm-3 mb-2"><input name="otherTopic" class="form-control" type="text" id="other-topic" placeholder="Other topic"></div>
+                            <div style="width: 100%; margin-top: 3%"> 
+                                <label for="book-list-text">Books:</label> 
+                                <p>Please enter a comma separated list of books with the title and author. <br>
+                                    Example: <br>
+                                    The Catcher in the Rye by J. D. Salinger, <br>
+                                    The Great Gatsby by F. Scott Fitzgerald, <br>
+                                    The Bluest Eye by Toni Morrison
+                                <textarea name="books" class="form-control" id="book-list-text" rows="8" required="" aria-reqiured=true></textarea>
+                            </div>
+                            <button class="btn" type="submit" id="new-list-submit">Submit</button>
+                        </form>
+                    </div>
+                    <div class="modal-footer">
+                        <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+                    </div>
+                    </div>
+                </div>
+            </div>
+        </div>        
+
+        <footer class="container-fluid text-center bg-blue">
+            <a href="#top" title="To Top">
+                <span id="top-arrow" class="glyphicon glyphicon-chevron-up"></span>
+            </a>
+            <p>Bootstrap Theme Made By W3Schools</a></p>
+        </footer>
```

Comment:
If we want the footer to have that curved aspect that the other pages have, we should add a div with class bg-blue-description with a div of class bg-white inside it.

Reply:
Ok, sounds good. I don't think the search page has the curve either, so I'll update that too.